### PR TITLE
Closes AgileVentures/MetPlus_tracker#78

### PR DESCRIPTION
**Pull request by amba178**

### CODE REVIEW

File: spec/models/company_role_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
```
Comment (joaopapereira) [inline spec/models/company_role_spec.rb]: Please have a consistent indentation on the files, or else it will be hard to look at them

**Comment by joaopapereira** [comment]
Please review my previous comment

**Comment by amba178** [comment]
I have fixed the indentations. it looked horrible before.

**Comment by joaopapereira** [comment]
Looks good

**Comment by amba178** [comment]
Moved to DONE!. already merged.

**Pull request merged**